index.html

```diff
@@ -45,6 +45,133 @@ <h2 class="tx-center">Full-Stack Software Developer</h2>
     <div class="blur-circle"></div>
   </section>
   <!--headline section-->
+  <hr>
+  <!--Documentation section-->
+  <section class="container container-wrap" id="documentation">
+    <div class="c-container ai-start">
+      <h6 class="h6-documentation">Languajes</h6>
+      <ul class="ul-documentation ul-base">
+        <li class="li-documentation">Javascrript</li>
+        <li class="li-documentation">Rubi</li>
+        <li class="li-documentation">HTML</li>
+        <li class="li-documentation">CSS</li>
+      </ul>
+    </div>
+    <div class="c-container ai-start">
+      <h6 class="h6-documentation">Frameworks</h6>
+      <ul class="ul-documentation ul-base">
+        <li class="li-documentation">Bootstrap</li>
+        <li class="li-documentation">Ruby on Rails</li>
+        <li class="li-documentation">RSPec</li>
+        <li class="li-documentation">CapyBara</li>
+        <li class="li-documentation">Selenium</li>
+      </ul>
+    </div>
+    <div class="c-container ai-start">
+      <h6 class="h6-documentation">Skills</h6>
+      <ul class="ul-documentation ul-base">
+        <li class="li-documentation">Codekit</li>
+        <li class="li-documentation">Github</li>
+        <li class="li-documentation">Codepen</li>
+        <li class="li-documentation">Gitlab</li>
+        <li class="li-documentation">Terminal</li>
+      </ul>
+    </div>
+  </section>
+  <!--Documentation section-->
+  <hr>
+  <div class="j-content">
+    <img class="icons" src="img/github-icon.svg" alt="Github logo">
+    <img class="icons" src="img/two-fingers-up.svg" alt="hand oing peace and love">
+    <img class="icons" src="img/twitter-icon.svg" alt="twitter logo">
+    <img class="icons" src="img/medium-icon.svg" alt="medium logo">
+  </div>
+  <div class="blur-circle bc-small"></div>
+  <!--My Work section-->
+  <section id="myWork">
+    <div class="j-content">
+      <ion-icon name="arrow-down-outline"></ion-icon>
+      <h5>WORK</h5>
+    </div>
+    <h3>MY RECENT<br>WORK</h3>
     <hr>
-
-</body>
\ No newline at end of file
+    <div class="g-container">
```

Comment:
- [ ] Please make sure each card item is laid out using `display: flex` property.